research/src/sources/fast.json5

```diff
@@ -72,7 +72,17 @@
     },
     {
       "name": "Menu",
-      "url": "https://explore.fast.design/components/fast-menu"
+      "url": "https://explore.fast.design/components/fast-menu",
+      "concepts": [
+        {
+          "name": "basic",
+          "image": "fast-menu-basic.png"
+        },
+        {
+          "name": "section",
+          "image": "fast-menu-section.png"
+        }
+      ]
```

Comment:
There's quite a few concepts not included here such as nested menus, start/end slots for icons and more, etc. Can we add those as research so we have a clear accounting of feature support?

Reply:
Fast submenus are represented in research in [97c0be4](https://github.com/openui/open-ui/pull/249/commits/97c0be4a5610957f716029c019c86a0ae7657765)